chat/chat.py

```diff
@@ -593,3 +593,9 @@ def _expand_static_url(self, url):
             except ImportError:
                 pass
         return url
+
+    @XBlock.handler
+    def chat_complete(self, request, suffix=""):
+        """This is called from the front end when the learner has completed the chat."""
+        # Does nothing at the moment
```

Comment:
Maybe mention that "Does nothing at the moment; this HTTP request is listened for by some mobile apps to trigger events after the chat is completed."